lib/main_common.pm

```diff
@@ -1760,10 +1758,22 @@ sub wicked_init_locks {
 
 sub load_extra_tests_wicked {
     wicked_init_locks();
-    my $ctx = wicked::TestContext->new();
-    for my $test (get_wicked_tests()) {
-        loadtest($test, run_args => $ctx);
-    }
+    my $ctx   = wicked::TestContext->new();
+    my @tests = get_wicked_tests();
+        for my $test (@tests) {
+            if (get_required_var('IS_WICKED_REF')) {
+                $test =~ s/\/sut\//\/ref\//;
```

Comment:
Only an idea:
```suggestion
                $test =~ s'/sut/'/ref/' if (get_required_var('IS_WICKED_REF'));
```
And you can remove the outer `if (get_required_var('IS_WICKED_REF')) {}` block.